basketball_reference_web_scraper/parsers/box_scores/play_by_play.py

```diff
@@ -0,0 +1,38 @@
+import re
+from lxml import html
+
+from basketball_reference_web_scraper.data import Location
+
+
+TIME_REGEX = "([0-9]+):([0-9]+)\.([0-9]+)"
+SCORE_REGEX = "([0-9]+)-([0-9]+)"
+
+
+def parse_play_by_plays(page):
+    tree = html.fromstring(page)
+    table = tree.xpath('//table[@id="pbp"]')
+    quarter = 0
+    result = []
+    for row in table[0][1:]:
+        if row[0].get("colspan") == "6":
```

Comment:
Some comments here might be nice about what the structure of the HTML is / what these values mean

Reply:
https://github.com/Yotamho/basketball_reference_web_scraper/blob/d270e618618feb17af3af435cecc1f87aa596590/basketball_reference_web_scraper/parsers/box_scores/play_by_play.py#L25-L30